src/DropdownMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Person } from './types/Person';
+
+interface Props {
+  people: Person[];
+  onSelected: (person: Person) => void;
+}
+
+export const DropdownMenu: React.FC<Props> = ({
+  people,
+  onSelected,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length > 0
+          ? (
+            people.map((person) => (
+              <div className="dropdown-item" key={person.slug}>
+                {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events */}
```

Comment:
let's try to fix this eslint issue. you give `onClick`  prop to paragraph, I think its not a good idea.